### PR TITLE
Ltd 3242 beis nuclear assessment flow

### PR DESCRIPTION
**Pull request by antroy-madetech**
Stay on Assessment page when submitting products.
If all products have been assessed, stay on Assessment page but show "Make recommendation" button.

### CODE REVIEW

File: caseworker/advice/templates/advice/assessed_products.html [hunk 1]
```diff
@@ -3,6 +3,9 @@
     <div class="govuk-grid-column-full">
         <table id="assessed-products" class="govuk-table">
             <h2 class="govuk-heading-m">Assessed products</h2>
+            {% if can_advise and buttons.make_recommendation %}
+                <a draggable="false" class="govuk-button govuk-button--primary govuk-!-margin-right-2" href="{% url 'cases:select_advice' queue_pk case.id %}">Make recommendation</a>
+            {% endif %}
```
Comment (saruniitr) [inline caseworker/advice/templates/advice/assessed_products.html:8]: We should not be showing "Make recommendation" button in this template.
It is already show in https://github.com/uktrade/lite-frontend/blob/master/caseworker/advice/templates/advice/view-advice.html#L39 when certain conditions are met. In this case when all trigger list products are reviewed.

File: caseworker/advice/views.py [hunk 2]
```diff
@@ -623,7 +623,7 @@ class BEISProductAssessmentView(AdviceView, BEISNuclearMixin, BEISAssessmentBase
     form_class = BEISTriggerListAssessmentForm
 
     def get_success_url(self):
-        return reverse("cases:advice_view", kwargs=self.kwargs)
+        return self.request.path
```
Comment (saruniitr) [inline caseworker/advice/views.py:626]: We only need to stay in this page until all products are reviewed.
Once all are reviewed then we need to above page where "Make recommendation" is made available to give advice.

File: caseworker/advice/templates/advice/assessed_products.html (hunk 1 above)
Comment (antroy-madetech) [reply caseworker/advice/templates/advice/assessed_products.html:8]: See the ticket. They now want the "Make recommendation" button here as well.

File: caseworker/advice/views.py (hunk 2 above)
Comment (antroy-madetech) [reply caseworker/advice/views.py:626]: The ticket says: "Once all products are reviewed you are directed to a ‘Make a recommendation’ screen, we would like them to be directed to the Assessed Products table with the ‘Make recommendation’ and ‘Clear assessments’ options"

**Comment by antroy-madetech** [comment]
Rework done on a new branch

**Pull request closed without merging**